packages/gatsby-theme-try-ghost/src/styles/screen.css

```diff
@@ -2092,7 +2092,7 @@ html.casper .post-full-content .kg-width-full .kg-image {
 }
 
 html.casper .post-full-content figure {
-    margin: 0.8em 0 2.3em;
+    margin: 0 0 2.3em;
```

Comment:
The original Ghost styles should remain intact here, see [original](https://github.com/TryGhost/Casper/blob/006b056adc6256847dce7ae4cae25015a96af500/assets/css/screen.css#L2095)